src/CONST.ts

```diff
@@ -853,7 +853,7 @@ const CONST = {
     // It's copied here so that the same regex pattern can be used in form validations to be consistent with the server.
     VALIDATE_FOR_HTML_TAG_REGEX: /<([^>\s]+)(?:[^>]*?)>/g,
 
-    VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX: /<([\s]+[\s\w~!@#$%^&*(){}[\];':"`|?.,/\\+\-=<]+.*[\s]*)>/g,
+    VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX: /<([\s]+.+[\s]*)>/g,
 
     WHITELISTED_TAGS: [/<>/, /< >/, /<->/, /<-->/, /<br>/, /<br\/>/],
```

Comment:
Can you fix these too? They seem to be missing ^ and $.
```suggestion
    WHITELISTED_TAGS: [/^<>$/, /^< >$/, /^<->$/, /^<-->$/, /^<br>$/, /^<br\/>$/],
```

Reply:
If we add `^$`, then it will fail to match `<>a`.

https://github.com/Expensify/App/blob/fe49e4f7fb9677a5834b53cb776f79fb20f893ce/src/components/Form/FormProvider.js#L145-L157

It will return false in line 146, but eventually, it will be true inside the loop though.

Reply:
Lets leave this then